webpack/scenes/ContentViews/ContentViewsActions.js

```diff
@@ -15,4 +17,23 @@ const getContentViews = extraParams => get({
   params: createContentViewsParams(extraParams),
 });
 
+const cvSuccessToast = (response) => {
+  const { data: { name } } = response;
+  return __(`Content view ${name} created`);
+};
+
+const cvErrorToast = (error) => {
+  const messages = getResponseErrorMsgs(error.response);
+  return __(messages);
```

Comment:
I don't think translate works dynamically, though I haven't come to a conclusive answer, generally I use it when there are fixed strings.

Error messages might be translated on the server end? :thinking: